static/app/utils/analytics/discoverAnalyticsEvents.tsx

```diff
@@ -11,7 +12,7 @@ export type DiscoverEventParameters = {
   'discover_v2.quick_context_header_copy': {clipBoardTitle: string};
   'discover_v2.quick_context_hover_contexts': {
     contextType: ContextType;
-    eventType?: 'error' | 'transaction';
+    eventType?: EventOrGroupType;
```

Comment:
I want to find a better type for this because it's including a few types that I don't know the use cases for 🤔